api/comments_api.py

```diff
@@ -24,28 +32,28 @@
 from internals import slo
 
 
-def amendment_to_json_dict(amendment: Amendment) -> dict[str, Any]:
-  return {
-      'field_name': amendment.field_name,
-      'old_value': amendment.old_value.strip('[]'),
-      'new_value': amendment.new_value.strip('[]'),
-      }
+def amendment_to_json_dict(amendment: Amendment) -> AmendmentModel:
```

Comment:
This still seems to return `dict[str, Any]` or more precisely `dict[str, str]`.  You could probably make it return `AmendmentModel` and I think that would work with `ActivityModel`.

Reply:
That would work but requires changing the test file too. I'll push some changes and see if that is what we want